scene/property_utils.cpp

```diff
@@ -65,6 +65,17 @@ Variant PropertyUtils::get_property_default_value(const Object *p_object, const
 		*r_is_valid = false;
 	}
 
+	// Handle special case "Script" property
+	if (p_property == SNAME("script")) {
+		if (r_is_valid) {
+			*r_is_valid = true;
+		}
+		Ref<Script> ct_scr = p_object->get_custom_type_script();
+		if (ct_scr.is_valid()) {
```

Comment:
```suggestion
		Ref<Script> ct_scr = p_object->get_custom_type_script();
		if (ct_scr.is_valid()) {
			if (r_is_valid) {
				*r_is_valid = true;
			}
```
I think we should only set `r_is_valid` if we are returning something here